react/index.js

```diff
@@ -1,21 +1,22 @@
-// Components
-export { default as StyleGuideProvider } from './StyleGuideProvider/StyleGuideProvider';
 export { default as SeekApp } from './StyleGuideProvider/StyleGuideProvider';  // Alias for backwards compatibility
+export { default as StyleGuideProvider } from './StyleGuideProvider/StyleGuideProvider';
```

Comment:
Haha, sorry, but you you mind flipping these two imports around? It makes for a bit of a pointless diff here 😄